meshroom/ui/app.py

```diff
@@ -352,5 +355,6 @@ def _default8bitViewerEnabled(self):
     pipelineTemplateFilesChanged = Signal()
     recentProjectFilesChanged = Signal()
     pipelineTemplateFiles = Property("QVariantList", _pipelineTemplateFiles, notify=pipelineTemplateFilesChanged)
+    pipelineTemplateNames = Property("QVariantList", _pipelineTemplateNames, notify=pipelineTemplateFilesChanged)
```

Comment:
I don't think this new property is needed, you could directly use `pipelineTemplateFiles` and avoid duplicating data

Reply:
It is not needed per se, but I feel like it simplifies greatly the parsing on the QML side: https://github.com/alicevision/Meshroom/pull/1833/commits/8bd4b6e04751d3425cb7373968631ed054f65741 
The parsing was, admittedly, not as efficient as it could have been but I still think there's a benefit in having `pipelineTemplateFiles`.